SPECS-SIGNED/kernel-hci-signed/kernel-hci-signed.spec

```diff
@@ -0,0 +1,153 @@
+%global debug_package %{nil}
+%global sha512hmac bash %{_sourcedir}/sha512hmac-openssl.sh
+%global buildarch x86_64
+%define uname_r %{version}-%{release}
+Summary:        Signed Linux Kernel for HCI
+Name:           kernel-hci-signed-%{buildarch}
+Version:        5.15.79.1
```

Comment:
I think we should also add a new entry to `version_release_matching_groups` inside `toolkit/scripts/check_entangled_specs.py` to make sure the regular and "signed" specs are updated in lockstep.

Reply:
The updates won't be blocked. Worst-case scenario someone will have to update the PR manually but that's still better than releasing kernels out of sync. Our build pipelines depend on these matching versions and I believe they would break during a release.

Reply:
@nicogbg and @rlmenge, what do you think is the best course of action?

Reply:
Automatic kernel upgrades include kernel-hci as well. @nicogbg can help with adding kernel-hci-signed to the group of kernels that get upgraded together.
@PawelWMS I am not sure what you are referring to here: 
'update the automation instead of manually creating the first "signed" spec.' I think kernel-hci-signed can be added to automation group and check_entangled_specs only after this PR is merged.
@vinceaperri please note that kernel-hci has been updated to v5.15.80
Thanks

Reply:
Ok. If I add this package to the automated upgrades and merge this PR on the same day, I think we can get away with adding it to entangle specs check.